cmd/cosign/cli/verify/verify_attestation.go

```diff
@@ -214,6 +228,17 @@ func (c *VerifyAttestationCommand) Exec(ctx context.Context, images []string) (e
 			}
 			co.SCT = sct
 		}
+	case c.TrustedRootPath != "":
+		if !c.NewBundleFormat {
+			return fmt.Errorf("unsupported: trusted root path currently only supported with --new-bundle-format")
+		}
+
+		// If a trusted root path is provided, we will use it to verify the bundle.
+		// Otherwise, the verifier will default to the public good instance.
```

Comment:
Do we need to explicitly called `FetchTrustedRoot` here and in `verify.go`? This is based on looking at https://github.com/sigstore/cosign/blob/b4be5f7a4305d195ee5a587e98903dfced269333/cmd/cosign/cli/verify/verify_blob.go#L403, where the PGI TUF metadata is loaded in for verify_blob/verify_blob_attestation before calling the internal verification function.

Reply:
Yes, good call! I just pushed a fix.